lib/jsdom/living/non-document-type-child-node.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const defineGetter = require("../utils").defineGetter;
+
+module.exports = function (core) {
+  // https://dom.spec.whatwg.org/#nondocumenttypechildnode
+
+  const implementers = [core.Element];
+  const ELEMENT_NODE = core.Node.ELEMENT_NODE;
+
+  implementers.forEach(function (Constructor) {
+    defineGetter(Constructor.prototype, "nextElementSibling", function () {
+      var element = this;
+      while ((element = element.nextSibling)) {
```

Comment:
Please don't implement these in terms of other public APIs. If you look at the performance characteristics of nextSibling, you can see that repeated calls to it will be very bad. Instead, use a similar strategy to what `nextSibling`'s code does.